traffic/TrafficGenerator.m

```diff
@@ -15,6 +15,7 @@
 		TrafficSource; 
 		AssociatedUeIds; 
 		ArrivalTimes;
+		TrafficSourceNoBackhaul;
```

Comment:
I think we should be careful about this. We're posing nested constraints and boundaries on class properties. Would it not make more sense to have the `TrafficSourceWithBackhaul`, and `TrafficSource` is then kept as the original traffic source. One would then have to select the traffic source based on the config, however, we could create a wrapper function in the Traffic generator than switches between objects. Then the traffic source property could also be moved to a private property (or protected)

Reply:
I'll do this in the backhaul branch.